lib/Basic/LaneBasedExecutionQueue.cpp

```diff
@@ -124,7 +127,7 @@ class LaneBasedExecutionQueue : public ExecutionQueue {
 #endif
 
     // Set the QoS class, if available.
-    setCurrentThreadQualityOfService(getDefaultQualityOfService());
+    setCurrentThreadQualityOfService(qos);
```

Comment:
🤔 not sure we actually want to change the QoS here.  I think we might want the lanes to run at default, and just launch the actual tasks at the specified level.

Reply:
OTOH, in process tasks would then run at a higher priority than spawned tasks.  So 🤷‍♂️